### PR TITLE
adding alias for SC

### PR DESCRIPTION
**Pull request by grodrigues3**
/assign @pwittrock

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -128,3 +128,11 @@ aliases:
   wg-resource-management-leads:
     - vishh
     - derekwaynecarr
+  steering-committee:
```
Comment (cblecker) [inline OWNERS_ALIASES]: This only includes the bootstrap members, and not the 6 elected members:
https://github.com/kubernetes/steering#members

**Comment by cblecker** [comment]
/hold
Also, this is going to get deleted by the next person who runs `make`.

**Comment by grodrigues3** [comment]
Whoops, forgot about the generator app.   Should I add the steering committee to sigs.yaml and regenerate things?

**Comment by cblecker** [comment]
It might make sense to define the steering committee in sigs.yaml, but they aren't really a sig or a wg, so we'd have to define another kind of construct. This might be a short term bandaid: https://github.com/kubernetes/community/pull/1849

**Comment by grodrigues3** [comment]
@cblecker fixed in the custom content section. that work?

**Comment by cblecker** [comment]
@grodrigues3 Yes, this should work.. however I messed something up in the verify script. Mind adding this?

```
diff --git a/hack/verify.sh b/hack/verify.sh
index 52d57a02..68d2fef5 100755
--- a/hack/verify.sh
+++ b/hack/verify.sh
@@ -11,7 +11,7 @@ function cleanup {
 }
 trap cleanup EXIT
 
-cp -r sigs.yaml sig-* wg-* Makefile generator ${WORKING_DIR}/
+cp -r sigs.yaml sig-* wg-* Makefile generator OWNERS_ALIASES ${WORKING_DIR}/
 
 cd ${WORKING_DIR}
 make 1>/dev/null
```

**Comment by cblecker** [comment]
This should verify properly once https://github.com/kubernetes/community/pull/1897 is merged.

**Pull request merged**